fr/contributors.md

```diff
@@ -0,0 +1,15 @@
+# Contributeurs
+Vous trouverez ici tous les contributeurs du projet. Si vous avez contribué au projet et n'êtes pas listé ici, merci de nous le faire savoir sur notre Github !
+
+## Mainteneurs et Développeur en chef
+- **Mainteneur:** [*Bombay*](https://github.com/BombayV)
+- **Mainteneur:** [*Guille*](https://github.com/guillerp8)
+- **Mainteneur/Community Manager:** [*Painless*](https://github.com/Pa1nless)
+- **Développeur en chef:** [*Germancito*](https://github.com/Germancitoz)
+
+## Membres de l'équipe
+- **Membre de l'équipe:** [*LittleFishy*](https://github.com/LittleFishyy)
```

Comment:
Move Fishy to contributors since he left the team.

Reply:
It's good